Pulumi.FSharp.Myriad/Pulumi.FSharp.Myriad.fsproj

```diff
@@ -39,9 +39,9 @@
     </ItemGroup>
     
     <ItemGroup>
-        <PackageReference Include="FsAst" Version="0.7.0" />
+        <PackageReference Include="FsAst" Version="0.13.0" />
```

Comment:
This upgrade breaks the build, it requires some extra work.

Reply:
Thanks; I have made the most of opening the project and I pushed the new packages. However, you're more than welcome to try and fix the compatibility with the new FsAst :)